### PR TITLE
Add anchor links to comments

### PR DESCRIPTION
**Pull request by asdil12**
https://progress.opensuse.org/issues/10720

### CODE REVIEW

File: assets/sass/theme_overwrites.scss [hunk 1]
```diff
@@ -118,6 +118,10 @@ a {
   @include transition(all 0.2s);
 }
 
+abbr[title], acronym[title] {
+  text-decoration: none;
+}
```
Comment (coolo) [inline assets/sass/theme_overwrites.scss]: huh?

**Comment by coolo** [comment]
please add a test case

File: assets/sass/theme_overwrites.scss (hunk 1 above)
Comment (asdil12) [inline assets/sass/theme_overwrites.scss]: we (bootstrap) set(s) our own dotted underline via border-bottom as chrome doesn't seem to support dotted underlines. This currently results in a single underline in chrome and a double underline in firefox.
This change will remove the double dotted line in firefox.

Comment (coolo) [inline assets/sass/theme_overwrites.scss]: Please reference https://github.com/twbs/bootstrap/issues/16574  as comment then

**Comment by asdil12** [comment]
Done.

**Comment by coolo** [comment]
https://github.com/os-autoinst/openQA/pull/910

**Pull request merged**